src/java.base/share/classes/java/util/concurrent/CopyOnWriteArrayList.java

```diff
@@ -1589,6 +1706,363 @@ public void forEachRemaining(Consumer<? super E> action) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Modifications to the reversed view are permitted and will be propagated
+     * to this list. In addition, modifications to this list will be visible
+     * in the reversed view. Sublists and iterators of the reversed view have
+     * the same restrictions as those of this list.
+     */
```

Comment:
```suggestion
     *
     * @since 21
     */
```
Per [8029241](https://bugs.openjdk.org/browse/JDK-8029241?focusedCommentId=14492604&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14492604), `@since` tags are not inherited at all and must be manually specified each time.